R/type_area.R

```diff
@@ -1,25 +1,27 @@
 #' @rdname type_ribbon
 #' @export
-type_area = function() {
+type_area = function(alpha = NULL) {
```

Comment:
Is it more transparent (and possible) to set the default to `alpha = tpar("ribbon.alpha")` in the function signature?